### PR TITLE
It's Party Time

### PR DESCRIPTION
**Pull request by Kurfursten**
@d3athrow 

I compiled this

### CODE REVIEW

**Comment by Irradiation** [comment]
>I didn't even compile this

Sasuga, c*llab-sama

**Comment by Kharkov72** [comment]
>compiling your code

what are you a fucking pussy

File: __DEFINES/dates.dm [hunk 1]
```diff
@@ -41,6 +41,7 @@
 #define BOXING_DAY "Boxing Day"
 #define NEW_YEARS_EVE "New Year's Eve"
 #define FRIDAY_THE_13TH "Friday the 13th"
+#define VG_BIRTHDAY "/vg/station Anniversary"
```
Comment (boy2mantwicethefam) [inline __DEFINES/dates.dm:44]: >capitalized VG

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -496,6 +496,12 @@
 	icon_state = "duncecap"
 	item_state = "duncecap"
 
+/obj/item/clothing/head/party_hat
+	name = "party hat"
+	desc = "A party cone. Not useful in disguise as a unicorn or impaling foes."
```
Comment (boy2mantwicethefam) [inline code/modules/clothing/head/misc.dm]: *Not useful to disguise as an unicorn or for impaling foes

File: code/modules/clothing/head/misc.dm [hunk 3]
```diff
@@ -496,6 +496,12 @@
...
+	icno_state = "birthdayhat"
```
Comment (Kharkov72) [inline code/modules/clothing/head/misc.dm]: oh no no no

Comment (boy2mantwicethefam) [inline code/modules/clothing/head/misc.dm]: What a fool you are. I'm a god, how can you kill a god? What a grand and intoxicating innocence. How could you be so naive? There is no escape. No recall or intervention can work in this place. Come, lay down your weapons, it is not too late for my mercy.

**Pull request merged**